### DISCUSSION TITLE
Stop uWS::App

### DISCUSSION THREAD

**Discussion by hellmannx** [uNetworking/uWebSockets#1340] [Q&A] [answered]
Is it possible to stop `uWS::App` after `run` function is called? I need to stop the server with a simple HTTP get request and have the program continue execution after where `run` gets called on `uWS::App` object.

    uWS::App app {};
    app.run(/* parameters */);
    // app.stop() ??
    /* Some other stufff */

**Comment by ghost** [comment, marked as answer]
https://github.com/uNetworking/uWebSockets/blob/master/misc/READMORE.md#apprun-and-fallthrough
